### DISCUSSION TITLE
Is there a function which can convert the RecurrenceRule and Start Time to Index value?

### DISCUSSION THREAD

**Discussion by cywan1998** [ical4j/ical4j#558] [General]
input: Recurrence rule, Recurrence Start Time, Occurrence RecurrenceId(Occurrence Start Time).
output: the index of the Occurrence in the recurrence series

**Comment by benfortuna** [comment]
Could you try using [Recur.getDates](http://ical4j.github.io/docs/ical4j/api/3.2.1/net/fortuna/ical4j/model/Recur.html#getDates(net.fortuna.ical4j.model.Date,net.fortuna.ical4j.model.Date,net.fortuna.ical4j.model.Date,net.fortuna.ical4j.model.parameter.Value)) with inputs: recurStartTime, occurrenceStartTime, etc.

You could then use `indexOf` on the returned [DateList](http://ical4j.github.io/docs/ical4j/api/3.2.1/net/fortuna/ical4j/model/DateList.html) to find the index of the occurence.
